src/Client/Features/Dashboard/Shared/RankedList.razor

```diff
@@ -12,7 +12,7 @@
     </MudItem>
     @foreach(var rankedItem in RankedItems.Skip(3))
     {
-        <MudItem xs=12 sm=6>
+        <MudItem xs=12 sm=6 Class="pa-0">
```

Comment:
I think we need to keep the left and right padding, otherwise, as your image shows, the cards stick out farther than the top 3.

Reply:
Good spot will amend

Reply:
This has been updated see image:

![Spacing2](https://user-images.githubusercontent.com/10053087/203844235-30cf7270-2047-47b4-adb8-03226685a5f2.png)